package.json

```diff
@@ -134,6 +134,17 @@
         "array"
       ],
       "angular/controller-as": "off",
+
+      "angular/module-setter": "warn",
+      "angular/module-getter": "warn",
+      "angular/typecheck-object": "warn",
+      "angular/typecheck-array": "warn",
+      "angular/typecheck-string": "warn",
+      "angular/no-private-call": "warn",
+      "max-len": "warn",
+      "promise/catch-or-return": "warn",
+      "promise/always-return": "warn",
+      
```

Comment:
These are the list of rules that have the most violations.  With the misc violations fixed, we can go through one by one to fix these.